warning: `QList<T> QSet<T>::toList()` is deprecated

```shell
/home/USER/Development/OpenMs/src/openms_gui/source/VISUAL/APPLICATIONS/TOPPASBase.cpp: In static member function ‘static OpenMS::TOPPASTreeView* OpenMS::TOPPASBase::createTOPPToolsTreeWidget(QWidget*)’:
/home/USER/Development/OpenMs/src/openms_gui/source/VISUAL/APPLICATIONS/TOPPASBase.cpp:487:52: warning: ‘QList<T> QSet<T>::toList() const [with T = QString]’ is deprecated: Use values() instead. [-Wdeprecated-declarations]
  487 |     QStringList category_list = category_set.toList();
      |                                 ~~~~~~~~~~~~~~~~~~~^~
In file included from /usr/include/qt/QtCore/qitemselectionmodel.h:45,
                 from /usr/include/qt/QtWidgets/qabstractitemview.h:46,
                 from /usr/include/qt/QtWidgets/qtreeview.h:44,
                 from /usr/include/qt/QtWidgets/qtreewidget.h:44,
                 from /usr/include/qt/QtWidgets/QTreeWidget:1,
                 from /home/USER/Development/OpenMs/src/openms_gui/include/OpenMS/VISUAL/TOPPASTreeView.h:41,
                 from /home/USER/Development/OpenMs/src/openms_gui/include/OpenMS/VISUAL/APPLICATIONS/TOPPASBase.h:43,
                 from /home/USER/Development/OpenMs/src/openms_gui/source/VISUAL/APPLICATIONS/TOPPASBase.cpp:35:
/usr/include/qt/QtCore/qset.h:265:14: note: declared here
  265 |     QList<T> toList() const { return values(); }
      |              ^~~~~~
```

--- a/src/openms_gui/source/VISUAL/APPLICATIONS/TOPPASBase.cpp
+++ b/src/openms_gui/source/VISUAL/APPLICATIONS/TOPPASBase.cpp
@@ -484,8 +484,10 @@ namespace OpenMS
     {
       category_set << String(it->second.category).toQString();
     }
+
     QStringList category_list(category_list.begin(), category_list.end());
-    qSort(category_list);
+    std::sort(category_list.begin(), category_list.end());
+
     std::map<QString, QTreeWidgetItem*> category_map;
 
     foreach(const QString &category, category_list)

--- a/src/openms_gui/source/VISUAL/APPLICATIONS/TOPPASBase.cpp
+++ b/src/openms_gui/source/VISUAL/APPLICATIONS/TOPPASBase.cpp
@@ -484,7 +484,7 @@ namespace OpenMS
     {
       category_set << String(it->second.category).toQString();
     }
-    QStringList category_list = category_set.toList();
+    QStringList category_list(category_list.begin(), category_list.end());
     qSort(category_list);
     std::map<QString, QTreeWidgetItem*> category_map;
 

--- a/src/openms_gui/source/VISUAL/APPLICATIONS/TOPPASBase.cpp
+++ b/src/openms_gui/source/VISUAL/APPLICATIONS/TOPPASBase.cpp
@@ -485,7 +485,7 @@ namespace OpenMS
       category_set << String(it->second.category).toQString();
     }
 
-    QStringList category_list(category_list.begin(), category_list.end());
+    QStringList category_list = category_set.values();
     std::sort(category_list.begin(), category_list.end());
 
     std::map<QString, QTreeWidgetItem*> category_map;

